Documentation/grids/grids.html

```diff
@@ -0,0 +1,161 @@
+            <a class="components" href="../alert/alert.html">Alerts</a>
+          </div>
+          <div class="item">
+            <a class="components" href="../badge/badge.html">Badge</a>
+          </div>
+          <div class="item">
+            <a class="components" href="../button/button.html">Button</a>
+          </div>
+          <div class="item">
+            <a class="components" href="../image/image.html">Image</a>
+          </div>
+          <div class="item">
+            <a class="components" href="../input/input.html">Input</a>
+          </div>
+          <div class="item">
+            <a class="components" href="../card/card.html">Card</a>
+          </div>
+          <div class="item">
+            <a class="components" href="../navigation/navigation.html"
+              >Navigation</a
+            >
+          </div>
+          <div class="item">
+            <a class="components" href="../rating/rating.html">Rating</a>
+          </div>
+          <div class="item">
+            <a class="components" href="../slider/slider.html">Slider</a>
+          </div>
+          <div class="item">
+            <a class="components" href="../text-utilities/text-utilities.html"
+              >Text-utilities</a
+            >
+          </div>
+          <div class="item">
+            <a class="components" href="../lists/lists.html">Lists</a>
+          </div>
+          <div class="item">
+            <a class="components" href="../modal/modal.html">Modal</a>
+          </div>
+          <div class="item">
+            <a class="components" href="../toast/toast.html">Toast</a>
+          </div>
+          <div class="item">
+            <a class="components" href="../grids/grids.html">Grids</a>
+          </div>
+        </div>
+      </div>
+      <div class="flex-col">
+        <div>
+          <h2>Two column grid</h2>
+          <div class="grid-2-2">
+            <div class="grid-item"></div>
+            <div class="grid-item"></div>
+            <div class="grid-item"></div>
+            <div class="grid-item"></div>
+          </div>
+        </div>
+        <div>
+          <iframe
+            src="https://carbon.now.sh/embed?bg=rgba%28171%2C+184%2C+195%2C+1%29&t=seti&wt=none&l=javascript&width=680&ds=true&dsyoff=20px&dsblur=68px&wc=true&wa=true&pv=56px&ph=56px&ln=false&fl=1&fm=Hack&fs=14px&lh=133%25&si=false&es=1x&wm=false&code=%253Ch2%253ETwo%2520column%2520grid%253C%252Fh2%253E%250A%253Cdiv%2520class%253D%2522grid-2-2%2522%253E%250A%253Cdiv%2520class%253D%2522grid-item%2522%253E%253C%252Fdiv%253E%250A%253Cdiv%2520class%253D%2522grid-item%2522%253E%253C%252Fdiv%253E%250A%253Cdiv%2520class%253D%2522grid-item%2522%253E%253C%252Fdiv%253E%250A%253Cdiv%2520class%253D%2522grid-item%2522%253E%253C%252Fdiv%253E%250A%253C%252Fdiv%253E"
+            style="
+              width: 400px;
+              height: 299px;
+              border: 0;
+              transform: scale(1);
+              overflow: hidden;
```

Comment:
Refrain from keeping styles in the html page. Keep them seperate instead.

Reply:
okay